x/oracle/keeper/single.go

```diff
@@ -181,3 +183,11 @@ func ResetCache() {
 func ResetAggregatorContextCheckTx() {
 	agcCheckTx = nil
 }
+
+func setCommonParams(p types.Params) {
```

Comment:
I do not like this approach. Why not use the database?

Reply:
These params only used when transactions are executed, and will not be changed frequently, so keep them in memory may be better.
They are initialized with AggregatorContext, to make sure they are initialized before any transaction be executed.
These variables can be moved to AggregatorContext, will do i when refactor the code